pkgs/development/python-modules/parse/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchPypi, fetchpatch
+, buildPythonPackage, python
+}:
+buildPythonPackage rec {
+  pname = "parse";
+  version = "1.6.6";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "71435aaac494e08cec76de646de2aab8392c114e56fe3f81c565ecc7eb886178";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "python-3.5-tests-compat.patch";
+      url = "https://github.com/r1chardj0n3s/parse/pull/34.patch";
+      sha256 = "16iicgkf3lwivmdnp3xkq4n87wjmr3nb77z8mwz67b7by9nnp3jg";
+    })
+  ];
+
+  checkPhase = ''
+    ${python.interpreter} test_parse.py
```

Comment:
@FRidh is there actually a reason `${python.interpreter}` is used all over the place in `python-modules` instead of just `python` which should be in `PATH`?

Reply:
Yes, because we have different interpreters, like `python3` (which nowadays have a symlink from `python`) and `pypy`.

Reply:
Are there cases where both interpreters end up in `buildPackages`?

Reply:
I hope not because then we would have issues also with `PYTHONPATH`.

In the end you need to be able to write `python test_parse.py` or `pypy test_parse.py` so I don't see any other way than using a variable like that here.